pkg/operator/staticpod/controller/installer/installer_controller_test.go

```diff
@@ -212,16 +212,13 @@ func TestNewNodeStateForInstallInProgress(t *testing.T) {
 	}
 
 	_, currStatus, _, _ = fakeStaticPodOperatorClient.GetStaticPodOperatorState()
-	if generation := currStatus.NodeStatuses[0].LastFailedRevision; generation != 2 {
-		t.Errorf("expected last failed revision generation for node to be 2, got %d", generation)
+	if generation := currStatus.NodeStatuses[0].LastFailedRevision; generation != 0 {
```

Comment:
this is an interesting side effect.  We don't mark the revision failed because we constantly retry it.  Objections/concerns?

Reply:
i'm ok with this as this might blew in unexpected ways if we start setting that property right?

also technically, if we still trying to run installer pod, retries means we are not "failed" (maybe we should after N retries?)